docker-compose.yml

```diff
@@ -57,4 +57,8 @@ services:
     environment:
       SCHEMA_REGISTRY_HOST_NAME: schema-registry
       SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'broker:29092'
-      SCHEMA_REGISTRY_LISTENERS: http://0.0.0.0:8081
\ No newline at end of file
+      SCHEMA_REGISTRY_LISTENERS: http://0.0.0.0:8081
+  prometheus-pushgateway:
+    image: prom/pushgateway:latest
+    ports:
+      - "9091:9091"
```

Comment:
Legg til en newline for å følge GH-konvensjoner

Reply:
Fikset